app/src/test/java/gov/va/vro/controller/MasControllerTest.java

```diff
@@ -140,13 +139,13 @@ void automatedClaimOutOfScope() throws Exception {
   void automatedClaimInScope() throws Exception {
 
     adviceWith(
-            camelContext,
```

Comment:
I've noticed this issue when doing these previously.  That is why I have wondered why we need both spotless AND checkstyle.  Maybe they check different things and this is one of the few things in common they both check, but it would be cool if we could stick to using one or the other if they do the same thing.

Reply:
I agree with you, however that is beyond the scope of this ticket